tachyon/math/polynomials/univariate/radix2_evaluation_domain_unittest.cc

```diff
@@ -0,0 +1,54 @@
+#include "tachyon/math/polynomials/univariate/radix2_evaluation_domain.h"
+
+#include <memory>
+
+#include "gtest/gtest.h"
+
+#include "tachyon/math/finite_fields/baby_bear/packed_baby_bear.h"
+#include "tachyon/math/finite_fields/koala_bear/packed_koala_bear.h"
+#include "tachyon/math/finite_fields/test/finite_field_test.h"
+#include "tachyon/math/matrix/matrix_types.h"
+#include "tachyon/math/polynomials/univariate/naive_batch_fft.h"
+
+namespace tachyon::math {
+
+namespace {
+
+template <typename F>
+class Radix2EvaluationDomainTest
+    : public FiniteFieldTest<
+          typename PackedPrimeFieldTraits<F>::PackedPrimeField> {};
+
+}  // namespace
+
+using RangeTypes = testing::Types<BabyBear, KoalaBear>;
+TYPED_TEST_SUITE(Radix2EvaluationDomainTest, RangeTypes);
+
+TYPED_TEST(Radix2EvaluationDomainTest, DFTBatch) {
```

Comment:
I'm not sure the test is being done correctly. Aren't they just calling the same function?

Reply:
They call Radix2EvaluationDomainTest's DFTBatch and Radix2EvaluationDomainTest's CosetLDEBatch test with F = BabyBear or F = KoalaBear separately. So a total of 4 tests are run.

Reply:
Oh I was thinking the test should check if the DFTBatch and the original FFT results in the same values but seems like this test isn't checking that. Am I correct?

Reply:
Yep this test checks against a naive implementation found in "naive_batch_fft.h"